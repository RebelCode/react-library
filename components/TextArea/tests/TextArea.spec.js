import React from "react";
import { render, cleanup } from "@testing-library/react";
import { TextArea } from "../index";

afterEach(cleanup);

describe("TextArea", () => {
  const Comp = <TextArea />

  it("should take a snapshot", () => {
    const { asFragment } = render(Comp);

    expect(asFragment(Comp)).toMatchSnapshot();
  });

  it("renders correctly", () => {
    const { queryByLabelText } = render(Comp);

    expect(queryByLabelText("TextArea")).toBeTruthy();
  });
});
