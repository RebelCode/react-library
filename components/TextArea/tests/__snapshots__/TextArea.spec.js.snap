// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea should take a snapshot 1`] = `
<DocumentFragment>
  <textarea
    aria-label="TextArea"
    data-testid="textarea"
  />
</DocumentFragment>
`;
